### PR TITLE
Variable rename to requirement

### LINKED ISSUE #322: Rename to requirements
image

- Rename the variable to requirements

### PR DESCRIPTION
**Pull request by AnjalineSingh**

### CODE REVIEW

**Comment by shahdivyank** [comment]
Have you tested your code?

**Comment by AnjalineSingh** [comment]
> 

yes

File: src/data/AdminForm.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const affiliations = [
   "marketing",
 ];
 
-export const requirementsData = {
+export const requirement = {
```
Comment (shahdivyank) [inline src/data/AdminForm.js:11]:
```suggestion
export const requirements = {
```

Comment (shahdivyank) [inline src/data/AdminForm.js:11]:
```suggestion
export const prerequisites = {
```

Comment (shahdivyank) [reply src/data/AdminForm.js:11]: could you update the import and then change the requirements, setRequirements to prerequisites, setPrerequisites

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**